### ISSUE TITLE
Support for Missing Zone

### ISSUE THREAD

**Issue by spriri** [ical4j/ical4j#511]
We are currently using ical4j 1.0.7 in which he zone America/Montreal is not available under zoneinfo so getTimeZone() is returning null. We had previously upgraded from 1.0.5.2 to 1.0.7. America/Montreal was available in 1.0.5.2 under zoneinfo in the jar, removed in 1.0.7 and was again added in the later releases of ical. As we don't want to upgrade or edit the code in the customer site is there any workaround to pull the missing zone in 1.0.7?

**Comment by benfortuna** [comment]
So to be clear, ical4j 1.0.7 was released 6 years ago (2015) and targeting JDK 1.4. I'm not sure which Java version you are running it on, but you really should consider upgrading to a modern Java version to avoid many security vulnerabilities that have been patched in later versions.

That said, I've had a look at the ical4j 1.x branch and added an alias for `/America/Montreal` -> `/America/Toronoto`. Note that I've also updated the target JDK to 1.6 (although even Java 6 shouldn't be used in production systems anymore).

Try release 1.0.8 to see if the issue is resolved, let me know of any issues.
